Add string pattern

alexec opened the pull request:
This will add the `pattern` for a string to the output.

alexec committed:
added pattern for string

alexec committed:
removed log

alexec committed:
updated rendering of pattern to match the value range style and format

nknapp commented:
Thanks for this PR. I'm not sure when I will find the time, but I will definitely look at it. I'd like to have a test-case but since there are no template-tests yet, I have to take care of the infrastructure first.

Would be cool if you could squash the commits. If I use the `squash-and-merge` button, you won't me mentioned as commit author anymore.

nknapp committed:
Slight modification of the "pattern"-look including test

- Patterns are now styled such that they read `must match /<pattern>/`
- Added test using the extracted test-setup from bootprint-openapi

nknapp committed:
Restyle like json-schema-enum

nknapp commented:
Closes #7
